src/guides/v2.4/graphql/mutations/reorder-items.md

```diff
@@ -0,0 +1,170 @@
+---
+group: graphql
+title: reorderItems mutation
+---
+
+The `reorderItems` mutation allows a logged-in user to add all the products from a previous order into their cart. The **Stores** > Settings > **Sales** > **Sales** > **Reorder** > **Allow Reorder** field must be set to **Yes** to enable reorders. You must provide a customer authorization token with the call.
+
+The mutation returns an error if it cannot add a product to the customer's cart:
+
+Error code | Description
+--- | ---
+INSUFFICIENT_STOCK | The requested quantity is greater than the available stock.
+NOT_SALABLE | The product is out of stock, disabled, or otherwise prevented from being sold.
+REORDER_NOT_AVAILABLE | Reorders have been disabled.
+PRODUCT_NOT_FOUND | The product has been deleted.
+UNDEFINED | An unknown error occurred.
+
+The `reorderItems` mutation will not add any products to the cart if it encounters the REORDER_NOT_AVAILABLE error. For any other error code, it will add the product to the cart, but it will continue adding other products, if applicable.
+
+{:.bs-callout-info}
+If any discounts or coupons from the original order are still valid, they will be applied to the cart.
```

Comment:
On reorder we just preparing shopping cart,  but discount coupon and gift cards can be applied only on shopping cart and checkout step. Other discount will be applied automatically if they are still actual.
It's better to remove this text

Reply:
Removed.